plugins/decklink/decklink-device-instance.cpp

```diff
@@ -639,20 +632,30 @@ HRESULT STDMETHODCALLTYPE DeckLinkDeviceInstance::VideoInputFormatChanged(
 		}
 	}
 
-	const HRESULT videoResult = input->EnableVideoInput(
-		displayMode, pixelFormat, bmdVideoInputEnableFormatDetection);
-	if (videoResult != S_OK) {
-		LOG(LOG_ERROR, "Failed to enable video input");
-		input->StopStreams();
-		FinalizeStream();
+	if (events & bmdVideoInputDisplayModeChanged) {
 
```

Comment:
Remove this line.

If you restructure code such that an the first line of an `if` statement's body is a blank line, you can remove that blank line.

Reply:
line removed